src/jdk.jfr/share/classes/jdk/jfr/internal/dcmd/DCmdDump.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2012, 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
You can't change the Oracle copyright notice. You can, and should, add an IBM copyright notice.
Any changes should first be proposed for the head stream (https://github.com/ibmruntimes/openj9-openjdk-jdk) and then back-ported as necessary to earlier versions.